web/cobrands/sass/_dashboard.scss

```diff
@@ -60,7 +60,6 @@
 
     canvas {
         width: 100% !important;
-        height: auto !important;
```

Comment:
I think you missed the word "height" out of your commit message for this one!

Reply:
Fixed!